lib/ansible/module_utils/network/meraki/meraki.py

```diff
@@ -127,6 +130,26 @@ def define_protocol(self):
         else:
             self.params['protocol'] = 'http'
 
+    def sanitize_keys(self, data):
+        if isinstance(data, dict):
+            items = {}
+            for k, v in data.items():
+                try:
+                    new = { self.key_map[k]: data[k] }
+                    items[self.key_map[k]] = self.sanitize_keys(data[k])
```

Comment:
`data[k]` is the same as `v`:
```suggestion
                    items[self.key_map[k]] = self.sanitize_keys(v)
```